includes/api/class-wc-rest-products-controller.php

```diff
@@ -141,12 +141,12 @@ public function query_args( $args, $request ) {
 
 		// Set up our temporary arguments to grab a product post.
 		$tmp_args = array(
-			'post_type' => 'products',
+			'post_type' => 'product',
 			'posts_per_page' => 1,
 		);
 
 		// Grab the first product.
-		$tmp_product = get_posts( $args )[0];
+		$tmp_product = get_posts( $tmp_args )[0];
```

Comment:
Still failing Travis on PHP 5.2 and 5.3

`Parse error: syntax error, unexpected '['`